demonstrations/tutorial_guide_to_pennylane_knowing_qiskit.py

```diff
@@ -0,0 +1,239 @@
+  dedicated teams for creating new demonstrations when we release new features, and an active
+  discussion forum for answering your questions.
+
+   
+"""
+
+######################################################################
+# In this demo, we're going to demonstrate to you the fundamentals of what makes PennyLane awesome with
+# the idea in mind that you're coming from Qiskit. If you want to follow along on your computer, you’ll
+# need to install the PennyLane-Qiskit plugin:
+# 
+# .. code-block:: 
+#
+#   pip install -U pennylane-qiskit
+#
+# Now, let’s get started.
+#
+
+######################################################################
+# PennyLane 🤝 Qiskit
+# -------------------
+#
+# With the first stable release of Qiskit in February 2024 — `Qiskit 1.0 <https://github.com/Qiskit/qiskit>`__ —
+# breaking changes were to be expected. Although learning a new language can be hard, PennyLane has very
+# simple tools that will help via the PennyLane-Qiskit plugin: your gateway to the land of PennyLane
+# that even lets you keep your existing Qiskit work without even having to know a ton about how PennyLane
+# works.
+#
+# There are two functions you need to know about:
+#
+# - :func:`~pennylane.from_qiskit`: converts an entire Qiskit ``QuantumCircuit`` — the whole thing — into
+#   PennyLane. It will faithfully convert Qiskit-side measurements (even mid-circuit measurements) or
+#   you can append PennyLane-side measurements directly to it.
+#
+# - :func:`~pennylane.from_qiskit_op`: converts a ``SparsePauliOp`` in Qiskit 1.0 to the equivalent operator in
+#   PennyLane.
+# 
+#
+# These two functions give you all that you need to access PennyLane’s features and user-interface
+# starting from the Qiskit side. As an example, let’s say you have the following code in Qiskit that
+# prepares a Bell state.
+#
+
+from qiskit import QuantumCircuit
+
+qc = QuantumCircuit(2)
+qc.h(0)
+qc.cx(0, 1)
+
+######################################################################
+# To convert this circuit to PennyLane, you just use ``qml.from_qiskit``:
+#
+
+import pennylane as qml
+
+pl_func = qml.from_qiskit(qc)
+print(qml.draw_mpl(pl_func, style="pennylane")())
+
+######################################################################
+# Want to measure some expectation values of Pauli operators, as well? Use ``qml.from_qiskit_op`` to
+# convert a ``SparsePauliOp`` into PennyLane’s equivalent operator.
+#
+
+from qiskit.quantum_info import SparsePauliOp
+
+qiskit_pauli_op = SparsePauliOp("XY")
+pl_pauli_op = qml.from_qiskit_op(qiskit_pauli_op)
+
+######################################################################
+# Then, you can *append* the expectation value measurement — done with ``qml.expval`` — to the PennyLane
+# circuit when you create it with ``qml.from_qiskit``:
+#
+
+pl_func = qml.from_qiskit(qc, measurements=[qml.expval(pl_pauli_op)])
+print(qml.draw_mpl(pl_func)())
```

Comment:
```suggestion
print(qml.draw_mpl(pl_func, style="pennylane")())
```
Not major, but just to match styles.